### ISSUE TITLE
Add / remove buttons should be buttons, not anchors

### ISSUE THREAD

**Issue by lynncyrin** [elo80ka/django-dynamic-formset#122]
This line https://github.com/elo80ka/django-dynamic-formset/blob/e10c0a22acc0c6152c9dfb95c4ba2f853502110e/src/jquery.formset.js#L187

(and a few other related lines)

Should be creating `button` elements instead of `a`. The reasoning behind this is that screenreaders and other HTML parsers assume that `a` elements will change the page location. This is why you have to add `href="javascript:void(0)"`

Buttons on the other hand, could be used to change the page contents without having to suppress a redirect

**Comment by andrewwong97** [comment]
I've had the same issue. Have you found a fix?

**Comment by lynncyrin** [comment]
I'm closing a bunch of my older github issues ♻️  feel free to open a new issue describing this same change! Or re-open this one, if someone is still interested in working on it 📝
